python/lsst/ts/ess/ess_csc.py

```diff
@@ -272,9 +291,10 @@ async def end_enable(self, id_data) -> None:
 
         self.log.info("Start periodic polling of the sensor data.")
         if self.local_mode:
-            await self.ess_instrument.start()
+            for ess_instrument in self.ess_instruments:
+                await ess_instrument.start()
         else:
-            await self.write(command="start", parameters={})
+            await self.write(command="start", parameters={})  # type: ignore
```

Comment:
This type: ignore should not be needed, and similarly for all other places you call `self.write`. What's going on?